server/src/main/java/org/elasticsearch/cluster/metadata/LifecycleExecutionState.java

```diff
@@ -81,7 +81,7 @@ public static Builder builder(LifecycleExecutionState state) {
             .setSnapshotName(state.snapshotName)
             .setShrinkIndexName(state.shrinkIndexName)
             .setSnapshotIndexName(state.snapshotIndexName)
-            .setRollupIndexName(state.downsampleIndexName)
```

Comment:
I took the opportunity to rename all instances of `rollup` to `downsample`. 

I know this adds a bit of noise to this PR, but I could not resist and did while I was in the area

Reply:
Thanks for doing this rename Christos (it's been quite confusing before)